src/Compilers/Core/Portable/Collections/ArrayBuilderExtensions.cs

```diff
@@ -249,5 +250,26 @@ public static void FreeAll<T>(this ArrayBuilder<T> builder, Func<T, ArrayBuilder
             }
             builder.Free();
         }
+
+#if COMPILERCORE
```

Comment:
Consider adding a comment what this condition mean and why we are using conditional compilation

Reply:
sure :)